src/app/containers/ATIAnalytics/canonical/index.jsx

```diff
@@ -1,10 +1,11 @@
 import React, { useEffect, useState } from 'react';
 import { string } from 'prop-types';
-import { atiBaseUrl } from '../atiUrl';
 import sendBeacon from '../../../lib/analyticsUtils/sendBeacon';
 
 const CanonicalATIAnalytics = ({ pageviewParams }) => {
-  const [atiPageViewUrl] = useState(atiBaseUrl + pageviewParams);
+  const [atiPageViewUrl] = useState(
+    process.env.SIMORGH_ATI_BASE_URL + pageviewParams,
```

Comment:
some might want this to be `${process.env.SIMORGH_ATI_BASE_URL}${pageviewParams}`, for the record, im not one of them :P